.fixtures.yml

```diff
@@ -6,9 +6,7 @@ fixtures:
   repositories:
     stdlib:
       repo: git://github.com/puppetlabs/puppetlabs-stdlib.git
-      ref: 4.13.1
```

Comment:
Sorry, my bad, I overlooked that you are referring to tags in git repos instead of referencing release on forge as I do in [one of my fixture files](https://github.com/hpc-unibe-ch/puppet-module-slurm/blob/main/.fixtures.yml). Using this format, the rspec-puppet tests are run against the latest available release of these dependencies.

In what direction do you want be to change this PR? Back to refs oder switch to forge releases?

Reply:
I think switching to forge release is fine.

Reply:
Ok, reworked towards using latest forge relases for both modules. As described in the commit message left the symlink to golang itself in terms of being explicit though not strictly necessary.